demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml

```diff
@@ -475,3 +475,11 @@ parameters:
     # This can be a relative or an absolute path, relative paths
     # will use DemosPlanPath::getRootDir() as base.
     deployment_sync_dir: '../htdocs'
+
+    # Allows to define links, that will be presented in a special section on the main page.
+    # Note that the data will be used in the frontend directly, i.e. no measures against potentially harmful
+    # scripts is taken if such thing is used here.
+    main_page_external_links:
+        DiPlan Portal: 'https://diplanportal-dev.demos-europe.eu/'
```

Comment:
Is the space intentional? Or should it be

```
        DiPlanPortal: 'https://diplanportal-dev.demos-europe.eu/'
```

Reply:
It is with empty space in Figma --> 
![image](https://user-images.githubusercontent.com/110987766/220872804-77749cee-e746-4354-928e-e798d4a93b02.png)

Is it a bad practice to add empty space in the variables in `yaml` file?

Reply:
While it is allowed via the spec, as this [so answer implies](https://stackoverflow.com/questions/40977183/spaces-in-yaml-keys), i asked myself if it is a good idea to use the key as the link text. But we can also change this later, when it poses an issue.